tasks/vendoring/__init__.py

```diff
@@ -743,9 +741,14 @@ def main(ctx, package=None, type=None):
     else:
         target_dirs = [vendor_dir, patched_dir]
     if package:
-        log("Using vendor dir: %s" % vendor_dir)
-        vendor(ctx, vendor_dir, package=package)
-        download_licenses(ctx, vendor_dir, package=package)
+        if type is None or type == "vendor":
```

Comment:
This is a change I was making time and time again to the vendoring script to actually respect the argument being passed.